com/uber/kafkaSpraynozzle/stats/StatsDReporter.java

```diff
@@ -0,0 +1,61 @@
+package com.uber.kafkaSpraynozzle.stats;
+
+import com.timgroup.statsd.NonBlockingStatsDClient;
+import com.timgroup.statsd.StatsDClient;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+/**
+ * Statsd StatsReporter implementation
+ *
+ * To initialize this class you need to provide it the path to a .properties file containing:
```

Comment:
I'm not a huge fan of this... if it was just the hostname:port it would be ok, but adding in the 3rd feels dirty.  It's a totally arbitrary string pattern that the clients would parse, and because we are giving this as an out-of-the-box options I feel like it should be as clean as possible.  We wont stop clients from doing similar things with their own stats/filters, but i don't want to set the precedent.